src/main/java/org/terasology/launcher/config/ConfigManager.java

```diff
@@ -95,10 +96,7 @@ private Config getDefaultConfigFor(final Path launcherDir) {
                 .dataDir(launcherDir.resolve("TerasologyData"))
                 .maxMemory(JavaHeapSize.GB_1_5)
                 .initMemory(JavaHeapSize.GB_1)
-                .javaParam("-XX:+UseParNewGC"
-                        + " -XX:+UseConcMarkSweepGC"
-                        + " -XX:MaxGCPauseMillis=20"
-                        + " -XX:ParallelGCThreads=10")
+                .javaParam(BaseLauncherSettings.USER_JAVA_PARAMETERS_DEFAULT)
```

Comment:
I guess this makes sense - I remember @praj-foss deliberately not using the `BaseLauncherSettings` anymore (in hope to completely switch to the new config management....)

Reply:
There are a lot of "using deprecated class BaseLauncherSettings" warnings :neutral_face: